app/src/main/java/com/github/se/project/ui/profile/CreateUserProfile.kt

```diff
@@ -128,6 +131,26 @@ fun CreateProfileScreen(
 
               Spacer(modifier = Modifier.weight(1f))
 
+              FirebaseMessaging.getInstance()
+                  .token
+                  .addOnCompleteListener(
+                      OnCompleteListener { task ->
+                        if (!task.isSuccessful) {
+                          // Log.w(TAG, "Fetching FCM registration token failed", task.exception)
+                          return@OnCompleteListener
+                        }
+
+                        // Get new FCM registration token
+                        token = task.result
+
+                        // Log and toast
+                        val msg = "FCM Token: $token"
+                        // Log.d(TAG, msg)
```

Comment:
Probably best to either remove commented-out logs rather than leave them lying around